### PR TITLE
Added style to FAQ page

### PR DESCRIPTION
**Pull request by Akshay-21**
Hey Hi @robogyan-adgitm i Added style for FAQ page. I have created PR kindly check it and merge.
Thank You!

### CODE REVIEW

File: dist/css/main.scss [hunk 1]
```diff
@@ -350,36 +368,38 @@ footer{
...
     }
 
-    .footer-container{
-        &-links{
-            li{
-                a{
+    .footer-container {
+        &-links {
+            li {
+                a {
```
Comment (robogyan-adgitm) [inline dist/css/main.scss:402]: All the changes in this file are spam

File: faq.html [hunk 2]
```diff
@@ -54,68 +55,438 @@ <h1>Robogyan</h1>
...
-  </div>
-  </p>
-      </div>
-</section>
-    
+    </section>
   </main>
+
+  <!-- Akshay Kumar Maurya -->
+  <!-- ================================================================== -->
```
Comment (robogyan-adgitm) [inline faq.html]: No personal info in code allowed

**Review by robogyan-adgitm — CHANGES_REQUESTED** [review]

File: dist/css/main.scss (hunk 1 above)
Comment (Akshay-21) [reply dist/css/main.scss:402]: Ok i will check this issue.

File: faq.html (hunk 2 above)
Comment (Akshay-21) [reply faq.html]: This one also i will check and remove this.

**Comment by Akshay-21** [comment]
Hey Hi @robogyan-adgitm after this comment i will see this and what do i need to do for this issue.

**Comment by robogyan-adgitm** [comment]
Just resolve these two things and recreate the PR

**Comment by Akshay-21** [comment]
> Just resolve these two things and recreate the PR

Hi @robogyan-adgitm, i have removed unused file and code and personal info, i have updated FAQ Accordion using smooth design with theme matching.

Please kindly check this.

Thank You!

image

File: dist/css/main.scss (hunk 1 above)
Comment (Akshay-21) [reply dist/css/main.scss:402]: Removed these unused things.
kindly resolve conversation.
Thank You!

File: faq.html (hunk 2 above)
Comment (Akshay-21) [reply faq.html]: Removed Personal info.
kindly resolve conversation.
Thank You!

**Review by robogyan-adgitm — APPROVED** [review]
Good

**Comment by robogyan-adgitm** [comment]
Not be merging this as from the design it's not clear that the unexpanded questions could be expanded. And for now current UI for the FAQs is better. Marked as hacktoberfest-accepted though as it is a good enough contribution to be worked on later.

**Comment by Akshay-21** [comment]
> Not be merging this as from the design it's not clear that the unexpanded questions could be expanded. And for now current UI for the FAQs is better. Marked as hacktoberfest-accepted though as it is a good enough contribution to be worked on later.

Thank You! for accepting the PR.

**Pull request closed without merging**